.github/workflows/ci.yml

```diff
@@ -0,0 +1,503 @@
+        run: |
+          ${{ env.TAS_SCRIPTS }}/start-compose.sh ${{ env.TAS_ENVIRONMENT }}/${{ matrix.compose-file }}
+          ${{ env.TAS_SCRIPTS }}/wait-for-alfresco-start.sh "http://localhost:8082/alfresco"
+      - name: "Run tests"
+        id: tests
+        timeout-minutes: ${{ matrix.deploy-timeout }}
+        run: mvn -B install -ntp -f ${{ matrix.pom-dir }}/pom.xml -P${{ matrix.profiles }} -Denvironment=default -DrunBugs=false
+      - name: "Print output after failure"
+        if: ${{ always() && steps.tests.outcome == 'failure' }}
+        run: ${TAS_SCRIPTS}/output_logs_for_failures.sh "${{ matrix.pom-dir }}"
+      - name: "Clean Maven cache"
+        run: bash ./scripts/ci/cleanup_cache.sh
+
+  tas-tests-with-aims:
+    name: ${{ matrix.testSuite }} TAS tests with AIMS
+    runs-on: ubuntu-latest
+    if: >
+      !contains(github.event.head_commit.message, '[skip tas]') &&
+      !contains(github.event.head_commit.message, '[skip tests]')
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - testSuite: REST API
+            pom: tests/tas-restapi/pom.xml
+            profiles: all-tas-tests,run-restapi-aims
+            deploy-timeout: 60
+          - testSuite: CMIS - BROWSER binding
+            pom: tests/tas-cmis/pom.xml
+            profiles: all-tas-tests,run-cmis-browser-with-aims
+            deploy-timeout: 40
+          - testSuite: CMIS - ATOM binding
+            pom: tests/tas-cmis/pom.xml
+            profiles: all-tas-tests,run-cmis-atom-with-aims
+            deploy-timeout: 40
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.30.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.30.0
+      - name: "Build"
+        timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
+        run: |
+          bash ./scripts/ci/init.sh
+          bash ./scripts/ci/build.sh
+      - name: "Set HOST_IP"
+        run: echo "HOST_IP=$(hostname  -I | cut -f1 -d' ')" >> $GITHUB_ENV
+      - name: "Set up the environment"
+        run: |
+          ${{ env.TAS_SCRIPTS }}/start-compose.sh ${{ env.TAS_ENVIRONMENT }}/docker-compose-aims.yml
+          ${{ env.TAS_SCRIPTS }}/wait-for-alfresco-start.sh "http://localhost:8082/alfresco" 180
+      - name: "Run tests"
+        id: tests
+        timeout-minutes: ${{ matrix.deploy-timeout }}
+        run: mvn -B install -ntp -f ${{ matrix.pom }} -P${{ matrix.profiles }} -Denvironment=aims-environment -DrunBugs=false "-Didentity-service.auth-server-url=http://${HOST_IP}:8999/auth"
+      - name: "Print output after failure"
+        if: ${{ always() && matrix.testSuite == 'REST API' && steps.tests.outcome == 'failure' }}
+        run: ${TAS_SCRIPTS}/output_logs_for_failures.sh "tests/tas-restapi"
+      - name: "Clean Maven cache"
+        run: bash ./scripts/ci/cleanup_cache.sh
+
+  cmis-tas-tests-elastic-search:
```

Comment:
`-` vs `_`, also `elasticsearch` is fine as one unique word.

Reply:
Done 👍